### PR TITLE
Store IDs for fake workloads to avoid overly reconciling

### PR DESCRIPTION
**Pull request by connorgorman**
## Description

Every time Central bounces or Sensor OOMs during a scale run then we end up reconciling every deployment and pod, which is unrealistic and ends up causing lots of churn for deployments, pods, alerts, network flows and processes. This should alleviate much of that by storing the active ids locally in Sensor and then recreate the active components based on that if there is a restart

```
sensor/service/pipeline/reconciliation: 2023/04/03 04:27:59.550878 reconciliation.go:148: Info: Reconciliation done. Number of objects deleted by type: map[*central.SensorEvent_Binding:397 *central.SensorEvent_ComplianceOperatorProfile:0 *central.SensorEvent_ComplianceOperatorResult:0 *central.SensorEvent_ComplianceOperatorRule:0 *central.SensorEvent_ComplianceOperatorScan:0 *central.SensorEvent_ComplianceOperatorScanSettingBinding:0 *central.SensorEvent_Deployment:0 *central.SensorEvent_Namespace:0 *central.SensorEvent_NetworkPolicy:0 *central.SensorEvent_Node:0 *central.SensorEvent_Pod:0 *central.SensorEvent_Role:0 *central.SensorEvent_Secret:0 *central.SensorEvent_ServiceAccount:0]
```

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

Will run a scale run

### CODE REVIEW

File: sensor/kubernetes/fake/fake.go [hunk 1]
```diff
@@ -123,7 +125,13 @@ func NewWorkloadManager(config *WorkloadManagerConfig) *WorkloadManager {
 		log.Panicf("could not unmarshal workload from file due to error (%v): %s", err, data)
 	}
 
+	db, err := pebble.Open("/var/cache/stackrox/pebble.db", &pebble.Options{})
+	if err != nil {
+		log.Panicf("could not open id storage")
+	}
```
Comment (lvalerom) [inline sensor/kubernetes/fake/fake.go:131]: Can we make this configurable? I think local-sensor will fail to start (unless run as root) if we try to start it with this path.

Comment (connorgorman) [reply sensor/kubernetes/fake/fake.go:131]: oh yeah, good point

Comment (connorgorman) [reply sensor/kubernetes/fake/fake.go]: Done, made it an env var that we apply to the scripts but not to local

**Review by dashrews78 — APPROVED** [review]
looks good from my perspective

File: sensor/kubernetes/fake/fake.go (hunk 1 above)
Comment (lvalerom) [reply sensor/kubernetes/fake/fake.go]: Awesome! 👍

**Review by lvalerom — APPROVED** [review]
LGTM. Thanks for adding the env variable.

**Pull request merged**